diff --git a/node_modules/docusaurus-plugin-matomo/src/index.js b/node_modules/docusaurus-plugin-matomo/src/index.js
index fe87ddd..8468fd5 100644
--- a/node_modules/docusaurus-plugin-matomo/src/index.js
+++ b/node_modules/docusaurus-plugin-matomo/src/index.js
@@ -9,15 +9,6 @@ module.exports = function (context) {
     throw new Error(`Please specify 'matomo' object in 'themeConfig' with 'matomoUrl' and 'siteId' fields in it to use docusaurus-plugin-matomo`);
   }
 
-  const {matomoUrl, siteId} = matomo;
-
-  if (!matomoUrl) {
-    throw new Error('Please specify the `matomoUrl` field in the `themeConfig.matomo`');
-  }
-  if (!siteId) {
-    throw new Error('Please specify the `siteId` field in the `themeConfig.matomo`');
-  }
-  const phpLoader = matomo.phpLoader || 'matomo.php';
   const jsLoader = matomo.jsLoader || 'matomo.js';
 
   const isProd = process.env.NODE_ENV === 'production';
@@ -26,7 +17,7 @@ module.exports = function (context) {
     name: 'docusaurus-plugin-matomo',
 
     getClientModules() {
-      return isProd ? [path.resolve(__dirname, './track')] : [];
+      return [];
     },
 
     injectHtmlTags() {
@@ -35,29 +26,13 @@ module.exports = function (context) {
       }
       return {
         headTags: [
-          {
-            tagName: 'link',
-            attributes: {
-              rel: 'preconnect',
-              href: `${matomoUrl}`,
-            },
-          },
           {
             tagName: 'script',
             innerHTML: `
-              var _paq = window._paq = window._paq || [];
-              _paq.push(['setRequestMethod', 'POST']);
-              _paq.push(['trackPageView']);
-              _paq.push(['enableLinkTracking']);
-              _paq.push(['enableHeartBeatTimer']);
-              (function() {
-                var u="${matomoUrl}";
-                _paq.push(['setRequestMethod', 'POST']);
-                _paq.push(['setTrackerUrl', u+'${phpLoader}']);
-                _paq.push(['setSiteId', '${siteId}']);
-                var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
-                g.type='text/javascript'; g.async=true; g.src=u+'${jsLoader}'; s.parentNode.insertBefore(g,s);
-              })();
+              var _mtm = window._mtm = window._mtm || [];
+              _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+              var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+              g.async=true; g.src='${jsLoader}'; s.parentNode.insertBefore(g,s);
             `,
           },
         ],
diff --git a/node_modules/docusaurus-plugin-matomo/src/track.js b/node_modules/docusaurus-plugin-matomo/src/track.js
deleted file mode 100644
index 719de54..0000000
--- a/node_modules/docusaurus-plugin-matomo/src/track.js
+++ /dev/null
@@ -1,14 +0,0 @@
-import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
-
-export default (function () {
-  if (!ExecutionEnvironment.canUseDOM) {
-    return null;
-  }
-  return {
-    onRouteUpdate({location}) {
-      _paq.push(['setCustomUrl', location.pathname]);
-      _paq.push(['setDocumentTitle', document.title]);
-      _paq.push(['trackPageView']);
-    },
-  };
-})();
